README.md

```diff
@@ -45,6 +45,12 @@ conda env create -f environment.yml
 conda activate pudl-cataloger
 ```
 
+## Setting up environment
+API tokens are required to interact with Zenodo. There is one set of tokens for accessing
+the sandbox server, and one for the production server. The archiver tool expects these tokens
+to be set in the following environment variables: `ZENODO_TOKEN_PUBLISH` and `ZENODO_TOKEN_UPLOAD`
+or `ZENODO_SANDBOX_TOKEN_PUBLISH` and `ZENODO_SANDBOX_TOKEN_UPLOAD` for the sandbox server.
+
```

Comment:
What's the difference between `publish` and `upload`?

Reply:
When I ran my update I only needed the `PUBLISH` keys (and I just replaced my upload keys with the word publish...)

Reply:
Upload allows you to upload new files to a deposition, but doesn't allow you to publish the deposition -- it could create a draft that a human has to publish.

Publish includes all the permissions of upload, but also allows publication.

Reply:
Are they supposed to be different keys though? I'm confused why my old upload keys worked as publish keys, also I couldn't run the `pudl_archiver` without the PUBLISH key but I could run it without the UPDATE key. Is that because you can run --sandbox without publish or --dry-run without publish?